core/src/main/java/org/mule/runtime/core/api/connectivity/ConnectivityTestingStrategy.java

```diff
@@ -21,7 +21,7 @@
 public interface ConnectivityTestingStrategy {
```

Comment:
I will move the ConnectionTestingService to the API and the exception types that it throws. But this one remains in core since I don't think we need to make it part of the mule-api since is used internally by the ConnectionTestingService default implementation.